src/components/Card/Card.tsx

```diff
@@ -45,7 +45,7 @@ export const Card = (props: CardI) => {
         <></>
       ) : (
         <div className={styles.corner_wrapper}>
-                <button className={styles.corner_button} onClick={props.setEditMode}>
+                <button className={styles.corner_button} onClick={EditPageClick}>
```

Comment:
**nitpick:** handlers should be named with `camelCase`, not `PascalCase`, so: `onClick={editPageClick}`

Reply:
Should be fixed now